designer/deployment-manager-api/src/test/scala/pl/touk/nussknacker/engine/api/deployment/OverridingProcessStateDefinitionManagerTest.scala

```diff
@@ -18,8 +18,8 @@ class OverridingProcessStateDefinitionManagerTest extends AnyFunSuite with Match
 
   private val defaultStateDefinitionManager: ProcessStateDefinitionManager = new ProcessStateDefinitionManager {
     override def stateDefinitions: Map[StatusName, StateDefinitionDetails] = Map(
-      DefaultState.name -> StateDefinitionDetails("Default", icon, None, Some("Default description")),
-      DefaultStateToOverride.name -> StateDefinitionDetails("Default to override", icon, None, Some("Default description to override"))
+      DefaultState.name -> StateDefinitionDetails("Default", icon, "", "Default description"),
```

Comment:
No no no, don't do this empty string.. :)

Reply:
Fixed.